src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java

```diff
@@ -36,12 +53,20 @@ public void toModelType_invalidPersonFile_throwsIllegalValueException() throws E
         assertThrows(IllegalValueException.class, dataFromFile::toModelType);
     }
 
-    //    @Test
-    //    public void toModelType_duplicatePersons_throwsIllegalValueException() throws Exception {
-    //        JsonSerializableAddressBook dataFromFile = JsonUtil.readJsonFile(DUPLICATE_PERSON_FILE,
-    //                JsonSerializableAddressBook.class).get();
-    //        assertThrows(IllegalValueException.class, JsonSerializableAddressBook.MESSAGE_DUPLICATE_PERSON,
-    //                dataFromFile::toModelType);
-    //    }
+    @Test
+    public void toModelType_duplicatePersons_throwsIllegalValueException() throws Exception {
+        JsonSerializableAddressBook dataFromFile = JsonUtil.readJsonFile(DUPLICATE_PERSON_FILE,
+                    JsonSerializableAddressBook.class).get();
+        assertThrows(IllegalValueException.class, JsonSerializableAddressBook.MESSAGE_DUPLICATE_PERSON,
+                    dataFromFile::toModelType);
+    }
+
+    @Test
+    public void toModelType_duplicateAppointments_throwsIllegalValueException() throws Exception {
+        JsonSerializableAddressBook dataFromFile = JsonUtil.readJsonFile(DUPLICATE_APPOINTMENT_FILE,
+                JsonSerializableAddressBook.class).get();
+        assertThrows(IllegalValueException.class, JsonSerializableAddressBook.MESSAGE_DUPLICATE_APPOINTMENTS,
+                dataFromFile::toModelType);
+    }
```

Comment:
great job including the tests that we left out previously - our code coverage is looking healthy